apps/mark/frontend/src/pages/diagnostics_screen.test.tsx

```diff
@@ -87,115 +100,23 @@ describe('System Diagnostics screen: Computer section', () => {
-      'Warning: The printer is low on toner. Replace toner cartridge.'
-    );
-    expectToHaveWarningIcon(warningText);
-    tonerLevelText = screen.getByText('Toner level: 2%');
-    expectToHaveWarningIcon(tonerLevelText);
-  });
-
-  it('shows a warning when the printer status cannot be loaded', async () => {
-    const hardware = MemoryHardware.buildStandard();
-    hardware.setPrinterIppAttributes({
-      state: 'unknown',
-    });
-    renderScreen({ hardware });
+    renderScreen();
 
     const printerStatusText = await screen.findByText(
-      'Could not get printer status.'
+      'Stopped - The printer is low on toner. Replace toner cartridge.'
     );
     expectToHaveWarningIcon(printerStatusText);
-
-    screen.getByRole('button', {
-      name: 'Refresh Printer Status',
-    });
-    screen.getByText('Last updated at 11:23 AM');
-  });
-
-  it('shows only the highest priority printer state reason', async () => {
-    const hardware = MemoryHardware.buildStandard();
-    hardware.setPrinterIppAttributes({
-      state: 'stopped',
-      stateReasons: [
-        'media-empty',
-        'marker-supply-low-report',
-        'door-open-warning',
-        'media-needed-error',
-      ],
-      markerInfos: [mockMarkerInfo()],
-    });
-    renderScreen({ hardware });
-
-    const warningText = await screen.findByText(
-      'Warning: The printer is out of paper. Add paper to the printer.'
-    );
-    expectToHaveWarningIcon(warningText);
-  });
-
-  it('shows the plain printer-state-reasons text for unrecognized printer state reasons', async () => {
-    const hardware = MemoryHardware.buildStandard();
-    hardware.setPrinterIppAttributes({
-      state: 'stopped',
-      stateReasons: ['some-other-reason-warning'],
-      markerInfos: [mockMarkerInfo()],
-    });
-    renderScreen({ hardware });
-
-    const warningText = await screen.findByText('Warning: some-other-reason');
-    expectToHaveWarningIcon(warningText);
-  });
-
-  it("doesn't show warning when printer-state-reasons can't be parsed", async () => {
-    const hardware = MemoryHardware.buildStandard();
-    hardware.setPrinterIppAttributes({
-      state: 'stopped',
-      stateReasons: ['123'],
-      markerInfos: [mockMarkerInfo()],
-    });
-    renderScreen({ hardware });
-
-    await screen.findByText('Printer status: Stopped');
-    expect(screen.queryByText(/Warning/)).not.toBeInTheDocument();
-  });
-
-  it("handles negative toner level (which indicates that the toner level can't be read)", async () => {
-    const hardware = MemoryHardware.buildStandard();
-    hardware.setPrinterIppAttributes({
-      state: 'idle',
-      stateReasons: ['none'],
-      markerInfos: [mockMarkerInfo({ level: -2 })],
-    });
-    renderScreen({ hardware });
-
-    const tonerLevelText = await screen.findByText('Toner level: Unknown');
+    const tonerLevelText = screen.getByText('Toner Level: 2%');
     expectToHaveWarningIcon(tonerLevelText);
   });
```

Comment:
@jonahkagan this is a case (negative toner levels) that I never handled in the print diagnostic for VVSG. Is this reproducible on our HP printers, or only applies to other printers?

Reply:
Unfortunately I have no memory of this, but I think I originally built this based on output from an HP printer, so it seems likely that's where I saw this behavior

Reply:
Making a note for myself: https://github.com/votingworks/vxsuite/issues/4854